openshift/che-monitoring.yaml

```diff
@@ -63,6 +63,48 @@ objects:
               memory: 256Mi
             requests:
               memory: 128Mi
+        - args:
+          - -provider=github
+          - -http-address=:3000
+          - -https-address=
+          - -htpasswd-file=/etc/proxy/htpasswd/auth
+          - -email-domain=*
+          - -upstream=${PROMETHEUS_SVC_URL}
```

Comment:
You either need to add this as a template parameter.

Declaration in the parameters section:
https://gitlab.cee.redhat.com/service/app-sre-observability/blob/master/openshift/prometheus.template.yaml#L177-178

And then set this value in the saas file once this PR is merged: https://github.com/openshiftio/saas-openshiftio/pull/1283